locales/id/app.po

```diff
@@ -648,6 +684,22 @@ msgctxt "screenshot_illustration"
 msgid "Capturing screenshot of entire page…"
 msgstr "Membuat tangkapan layar seluruh laman…"
 
+msgctxt "screenshot_view"
+msgid "view"
```

Comment:
This needs a comment to explain if it's a noun or verb. Is there a reason why these are lowercase?

Reply:
I'll add the comments later. They're lowercase because the component that is using these actions will automatically do all caps. So I prefer to leave them as lowercase ones to let the system decide what is the best way to do all caps for each locale.

Reply:
To be honest, that should be the other way around: don't rely on text transformation, put them all caps in the localization file.

Not sure about Android, but at least with Chrome and CSS, uppercase isn't bug free.

Reply:
That's a very useful piece of advice. I'll change them into all caps tomorrow and share this with the engineering team.